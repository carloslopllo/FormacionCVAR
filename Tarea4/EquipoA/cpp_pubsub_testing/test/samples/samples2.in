1 2.5 4.1 5.1 3.2 1.0 14 6 