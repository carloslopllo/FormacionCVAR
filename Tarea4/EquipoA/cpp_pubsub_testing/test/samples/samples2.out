1 1.750000
3.300000
4.600000
4.150000	2.100000
7.500000
	10.000000
